src/TodoList.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import './redux/Todo.css'
+import TodoItem from './TodoItem'
+import { useSelector } from 'react-redux'
+
+const TodoList = () => {
+     const todos = useSelector(state => state);
+    return (
```

Comment:
here it should be state => state.yourVariableInStore (state.todos or state.newTodos) based on how you are storing.

Reply:
to do that you need to return an object which is proper way.

Reply:
Here you are accessing in proper way. but in reducer for update item you are returning undefined